File: benchmark/results/ts-invocation-inline/run-001/b-baseline/output/src/domain/identity/services/AuthService.ts
import { User } from "../aggregates/User";

/** Service responsible for authenticating users. */
export class AuthService {
  create(input: Partial<User>): User {
    void input;
    throw new Error("not implemented");
  }

  findById(id: string): User | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): User[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}

File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/identity/services/auth_service.py
"""AuthService for the identity domain."""

from typing import Optional

from metaengine_demo.identity.aggregates.user import User


class AuthService:
    """Service responsible for authentication of users."""

    def create(self, input: Optional[User]) -> User:
        """Create authentication state for a new user."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[User]:
        """Find authentication state for a user by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[User]:
        """List authenticated users up to the given limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Revoke authentication for a user by id."""
        raise NotImplementedError("not implemented")
